src/validation/overseas.entity.query.validation.ts

```diff
@@ -0,0 +1,7 @@
+import { body } from "express-validator";
+
+import { ErrorMessages } from "./error.messages";
+
+export const overseasEntityQuery = [
+  body("oe_number").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.OE_QUERY_NAME).isLength({ min: 8, max: 8 }).withMessage(ErrorMessages.INVALID_OE_LENGTH),
```

Comment:
You might also need to check for invalid characters in this field - see other validation examples.

Reply:
it is extremely long line please sread by line
```

body("oe_number")
    .not().isEmpty({ ignore_whitespace: true })
    .withMessage(ErrorMessages.OE_QUERY_NAME)
    .isLength({ min: 8, max: 8 })
    .withMessage(ErrorMessages.INVALID_OE_LENGTH),
```